src/i18n/de/wallet.ts

```diff
@@ -8,7 +8,14 @@ const wallet: WalletMessages = {
   'wallet.column.name': 'Name - DE',
   'wallet.column.ticker': 'Ticker - DE',
   'wallet.column.balance': 'Balance - DE',
-  'wallet.column.value': 'Value - DE'
+  'wallet.column.value': 'Value - DE',
+  'wallet.action.send': 'Send - DE',
+  'wallet.action.receive': 'Receive - DE',
+  'wallet.transaction.table.type': 'Type - DE',
+  'wallet.transaction.table.address': 'Address - DE',
+  'wallet.transaction.table.to': 'To - DE',
+  'wallet.transaction.table.amount': 'Amount - DE',
+  'wallet.transaction.table.coin': 'Coin - DE'
```

Comment:
Most of these translations are very "common" and might be used at other places as well, such as `Name`, `Amount`, `Value` etc. Put translations into `wallet` only, if we don't need these at other places. Or move these translations into `i18n/{lang}/common`. 

We don't need it now, but we should keep that in mind.